src/lib/detection_algorithms/cntk/reviewer.js

```diff
@@ -0,0 +1,72 @@
+const path = require('path');
```

Comment:
Rename the folder from "cntk" to cntkfastrcnn of something similiar. e.g. CNTK might support more detection algorithms.